proj1a/ArrayDeque.java

```diff
@@ -45,12 +40,13 @@ private int nextL(int i) {
     private void resize(int capacity) {
         T[] temp = (T[]) new Object[capacity];
         int old_size = size;
```

Comment:
`old_size` is an unnecessary variable that made me think there was some logic depending on it.